src/api/public/apidocs-new/paths/search_project_id.yaml

```diff
@@ -28,10 +28,6 @@ get:
             - `group/@role`: name of the role which a group could be related with this project.
           - More complex predicates are: `maintenance/maintains/@project`, `repository/@name`,
             `repository/path/@project`, `repository/releasetarget/@trigger`.
-
-        These functions can also be used with predicates: `boolean`, `contains`, `not`, `starts_with`, `ends_with`.
```

Comment:
There is the same documentation in `GET /search/project`. To be consistent, we should also remove it.

Reply:
I removed the other mentions of this as well :+1: